examples/cryptorealtime/terraform-setup/main.tf

```diff
@@ -0,0 +1,152 @@
+provider "google" {
+ region = "${var.region}"
+ credentials = "${file("${var.credsfile}")}"
+ project="${var.project_id}"
+}
+
+
+/*
+resource "google_project_service" "iamapi" {
+  project ="${var.project_id}"
+  service ="iam.googleapis.com"
+  disable_on_destroy = false
+
+}
+
+resource "google_project_service" "bigtableapi" {
+    project ="${var.project_id}"
+    service ="bigtableadmin.googleapis.com"
+    disable_on_destroy = false
+}
+
+resource "google_project_service" "computeapi" {
+    project ="${var.project_id}"
+    service ="compute.googleapis.com"
+    disable_on_destroy = false
+}
+
+resource "google_project_service" "dataflowapi" {
+    project ="${var.project_id}"
+    service ="dataflow.googleapis.com"
+    disable_on_destroy = false
+}
+
+resource "google_project_service" "storageapi" {
+    project ="${var.project_id}"
+    service ="storage.googleapis.com"
+    disable_on_destroy = false
+}
+
+
+*/
+resource "google_bigtable_instance" "instance" {
+  project = "${var.project_id}"
+  name         = "${var.bigtable_instance_name}"
+  instance_type = "PRODUCTION"
+  cluster {
+    cluster_id   = "${var.bigtable_instance_name}-cluster"
+    zone = "${var.zone}"
+    num_nodes    = 1
+    storage_type = "HDD"
+  }
+}
+
+ 
+resource "google_compute_instance" "default" {
+ project = "${var.project_id}"
```

Comment:
Consistent indenting please. 2 spaces is good. 4 spaces is fine. 1/2/4 in different places is bad.